ms2deepscore/SettingsMS2Deepscore.py

```diff
@@ -132,13 +102,30 @@ class GeneratorSettings:
             Number of pairs for each InChiKey14 during each epoch. Default=1
         shuffle
             Set to True to shuffle IDs every epoch. Default=True
-        ignore_equal_pairs
+        ignore_equal_pairs TODO: check against include_diagonal
```

Comment:
Yes I think this is the same parameter as include_diagonal. But fine to change later.